core/src/main/java/org/apache/iceberg/rest/ErrorHandlers.java

```diff
@@ -297,7 +297,9 @@ public void accept(ErrorResponse error) {
           throw new ServiceUnavailableException("Service unavailable: %s", error.message());
       }
 
-      throw new RESTException("Unable to process: %s", error.message());
+      throw new RESTException(
+          "Unable to process (code: %s, type: %s): %s",
+          error.code(), error.type() != null ? error.type() : "unknown", error.message());
```

Comment:
i wonder if making the null as unknown adds more value ? how about just having error.type() ?